content/community/chat.md

```diff
@@ -9,26 +9,29 @@ aliases:
 
 The XSF hosts a variety of chat rooms and all archives are available online. Participation in the XSF's discussion channels is subject to the [Community Code of Conduct](https://xmpp.org/extensions/xep-0458.html).
 
-### Archives
```

Comment:
This should stay ###, since it's about archives. The points following this chapter contain archive links, but are not exclusively about archives. They contain links to live chats.

Reply:
reverted e571b03